### ISSUE TITLE
Want --json output for forget.

### ISSUE THREAD

**Issue by irishburlybear** [restic/restic#3117]
Output of `restic version`
--------------------------
restic 0.11.0 compiled with go1.15.3 on windows/amd64

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------
output of forget should have an option for --json

repository a3aca485 opened successfully, password is correct
[0:00] 100.00%  1 / 1 files deleted

What are you trying to do? What problem would this solve?
---------------------------------------------------------
Trying to hook up stdout to a restic GUI I'm making.
Forget is pretty quick for most I assume, but for bigger 
repos, It would be nice to give some feedback to the user.
There is already a percentage and a file count. Just need it to 
be json so I can stream stdout the results.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
restic always impresses me. I'm just trying to make it more user friendly for the masses.
I'm currently making an electron app. I know everyone hates electron but I see it's the only way to
be platform agnostic.

**Comment by aawsome** [comment]
`forget` should already support the `--json`option.
Can you please give an example of a `forget --json` run: What output do you acutally get and what you want to get?

**Comment by fd0** [comment]
Indeed, `forget --json` already works. Why do you think it doesn't? :)

**Comment by irishburlybear** [comment]
I get no output using --json. It's just blank. Where if I leave off --json. I get some stdout. It's not like a big deal or anything, I have it displaying the summary as an alert.

**Comment by fd0** [comment]
Hm, can you show us what's printed on stdout? If nothing is forgotten, then restic won't print any json output...

**Comment by metalsp0rk** [comment]
I'm seeing no output on `restic forget --json`, output shown below:

```
$ ./restic snapshots -r ~/test_repo
repository a0989999 opened successfully, password is correct
ID        Time                 Host        Tags        Paths
------------------------------------------------------------------------------
75e03da4  2020-11-25 22:44:28  monolith                /home/$USER/Videos

5224a5e0  2020-11-25 22:44:56  monolith                /home/$USER/Pictures

c1983770  2020-11-25 22:46:11  monolith                /home/$USER/Videos

b80b8e20  2020-11-25 23:40:50  monolith                /home/$USER/Videos

955cad15  2020-11-25 23:40:55  monolith                /home/$USER/Videoss
------------------------------------------------------------------------------
5 snapshots
$ ./restic forget --json -r ~/test_repo 955cad15 | jq
$ ./restic snapshots -r ~/test_repo
repository a0989999 opened successfully, password is correct
ID        Time                 Host        Tags        Paths
------------------------------------------------------------------------------
75e03da4  2020-11-25 22:44:28  monolith                /home/$USER/Videos

5224a5e0  2020-11-25 22:44:56  monolith                /home/$USER/Pictures

c1983770  2020-11-25 22:46:11  monolith                /home/$USER/Videos

b80b8e20  2020-11-25 23:40:50  monolith                /home/$USER/Videos
------------------------------------------------------------------------------
4 snapshots

```

This is an example of normal output for the forget command:

```
$ ./restic forget  -r ~/test_repo d332874a
repository a0989999 opened successfully, password is correct
[0:00] 100.00%  1 / 1 files deleted
```

**Comment by aawsome** [comment]
Ah, I guess to point is the direct given snapshot id. 

`restic forget --json` gives the table with snapshots to delete/keep and the reasons for it - as does the command without `--json`.
If you give the ids to forget directly, `restic forget` simply removes the snapshots. Without json you also don't so that table but only the verbosity of the deletion process...

**Comment by stephan0307** [comment]
I am missing in the json output for forget the id's in the snapshots within  **reasons**.
For me it would be enough, when here only the id's would be included as a reference to **keep**.

I made for this requirement a local change and could push the change if wanted.
